src/main/java/org/amahi/anywhere/activity/RecentFilesActivity.java

```diff
@@ -263,12 +263,6 @@ public void onFileOptionSelected(FileOptionClickEvent event) {
         selectedFileOption = event.getFileOption();
         switch (selectedFileOption) {
             case FileOption.DOWNLOAD:
-                if (Android.isPermissionRequired()) {
```

Comment:
this is more than linting! ...

Reply:
Oh, it's because it so happens that the case under it is identical. Hmm..

Reply:
Yes! Identical cases.